src/components/MapView/MapView.website.tsx

```diff
@@ -253,20 +251,12 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
                     {directionCoordinates && <Direction coordinates={directionCoordinates} />}
                 </Map>
                 <View style={[styles.pAbsolute, styles.p5, styles.t0, styles.r0, {zIndex: 1}]}>
-                    <PressableWithoutFeedback
-                        accessibilityRole={CONST.ROLE.BUTTON}
+                    <Button
                         onPress={centerMap}
-                        accessibilityLabel={translate('common.center')}
-                    >
-                        <View style={styles.primaryMediumIcon}>
-                            <Icon
-                                width={variables.iconSizeNormal}
-                                height={variables.iconSizeNormal}
-                                src={Expensicons.Crosshair}
-                                fill={theme.icon}
-                            />
-                        </View>
-                    </PressableWithoutFeedback>
+                        iconFill={theme.icon}
+                        medium
+                        icon={Expensicons.Crosshair}
```

Comment:
I think we'd want to keep the `accessibilityLabel` if going down this route.

Reply:
@truph01 Bump on this :pray:

Reply:
@jjcoffee I updated it